meta-aos-rcar-gen4-dom0/recipes-core/runx/runx_git.bb

```diff
@@ -0,0 +1,60 @@
+HOMEPAGE = "https://github.com/lf-edge/runx"
+SUMMARY = "runx stuff"
+DESCRIPTION = "Xen Runtime for OCI"
+
+SRCREV_runx = "0c7edb3453398d7a0c594ce026c9c1e93c2541cc"
+
+SRC_URI = "\
+	  git://github.com/lf-edge/runx;nobranch=1;name=runx \
+	  file://0001-files-create-Update-create-script-to-start-unikernel.patch \
+	  "
+
+SRC_URI[md5sum] = "ce9b2d974d27408a61c53a30d3f98fb9"
+SRC_URI[sha256sum] = "bf338980b1670bca287f9994b7441c2361907635879169c64ae78364efc5f491"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=945fc9aa694796a6337395cc291ddd8c"
+
+S = "${WORKDIR}/git"
+PV = "v1.1-git${SRCREV_runx}"
+
+inherit pkgconfig
+
+RDEPENDS_${PN} += " jq bash"
```

Comment:
Combine two RDEPENDS, except for the case values are combined by some groups/topics etc. Also you don't need `+=` in the first case. Do you?

Reply:
Yes. I agree. I'll fix it.